aggregator_core/src/datastore.rs

```diff
@@ -1332,7 +1332,7 @@ impl<C: Clock> Transaction<'_, C> {
                         ON aggregation_jobs.id = report_aggregations.aggregation_job_id
                         WHERE aggregation_jobs.task_id = (SELECT id FROM tasks WHERE task_id = $1)
                     ) AS report_aggs
-                    ON report_aggs.client_report_id = client_reports.id
+                    ON report_aggs.client_report_id = client_reports.report_id  -- this join is very inefficient, fix before deploying in non-test scenario
```

Comment:
I see there is no index on client_reports.report_id, is this all that would be necessary?

Reply:
Possibly. As noted, this is test code; I'll just remove it instead in #1508.